sonar-project.properties

```diff
@@ -0,0 +1,6 @@
+sonar.organization=bean-pod
+sonar.projectKey=bean-pod_switchboard
+
+# relative paths to source directories. More details and properties are described
+# in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
+sonar.sources=.
```

Comment:
Could we explicitly specify the source code folders here, for the back-end and the front-end? Which would be
`/service/switchboardapp/src` and `frontend/src`. I want to avoid getting noise in the reports if sonarqube starts anaylzing random files like the pom.xml, package-lock.json, and others.

Reply:
Yes we can. As already stated, our repository should follow [Maven's standard directory layout](http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html)